src/contribute/contribution-guide.md

```diff
@@ -231,10 +235,54 @@ Then you can push your local, committed changes to your (forked) repository on G
 ### Testing
 All code should have appropriate unit testing coverage. New code should have new tests in the same contribution. Bug fixes should include a regression test to prevent the issue from reoccurring.
 
+#### Java SDK
+
 For contributions to the Java code, run unit tests locally via Maven. Alternatively, you can use Travis-CI.
 
     $ mvn clean verify
 
+#### Python SDK
+
+For contributions to the Python code, you can use following commands to run unit tests locally and to check for lint errors. Once you send a pull request for review, these tests will be rerun using Jenkins. We recommend setting up a virtual environment before testing your code.
```

Comment:
Maybe a TBD for people who wants to use `mvn`. FWIW `mvn clean verify` works in the python directory. It should work from the root with the right selection of project (`mvn clean verify -pl sdks/python`)